dags/analyzer_helper/tests/unit/test_load_transformed_data_base.py

```diff
@@ -23,18 +23,22 @@ def test_get_platform_id(self):
         """
         self.assertEqual(self.test_instance.get_platform_id(), self.platform_id)
 
-    @patch('from dags.analyzer_helper.discord.load_transformed_data_base.LoadTransformedDataBase')
+    @patch.object(LoadTransformedDataBase, 'load_impl')
```

Comment:
What's the `load_impl` object? I don't see it on your latest commit in https://github.com/TogetherCrew/airflow-dags/blob/feat/analyzer-helpers-discord/dags/analyzer_helper/discord/load_transformed_members_base.py